src/site/_data/postTags.js

```diff
@@ -0,0 +1,79 @@
+const tags = {
+  "3d": "3D",
+  accessibility: "Accessibility",
+  amp: "AMP",
+  analytics: "Analytics",
+  audio: "Audio",
+  "augmented-reality": "Augmented Reality",
+  canvas: "Canvas",
+  capabilities: "Capabilities",
+  "case-study": "Case Study",
+  cast: "Cast",
+  "chrome-ux-report": "Chrome UX Report",
+  "chrome-dev-summit": "Chrome Dev Summit",
+  "content-security-policy": "Content Security Policy",
+  cookies: "Cookies",
+  cors: "CORS",
+  css: "CSS",
+  devtools: "DevTools",
+  dom: "DOM",
+  encryption: "Encryption",
+  "feature-policy": "Feature Policy",
+  "file-system": "File System",
+  fonts: "Fonts",
+  forms: "Forms",
+  games: "Games",
+  graphics: "Graphics",
+  houdini: "Houdini",
+  identity: "Identity",
+  images: "Images",
+  install: "Install",
+  intl: "Intl",
+  iot: "IoT",
+  javascript: "JavaScript",
+  layout: "Layout",
+  lighthouse: "Lighthouse",
+  media: "Media",
+  "media-queries": "Media Queries",
+  mobile: "Mobile",
+  modules: "Modules",
+  monetization: "Monetization",
+  network: "Network",
+  node: "Node",
+  notifications: "Notifications",
+  offline: "Offline",
+  "origin-trials": "Origin Trials",
+  payments: "Payments",
+  performance: "Performance",
+  permissions: "Permissions",
+  "progressive-web-apps": "Progressive Web Apps",
+  puppeteer: "Puppeteer",
+  rendering: "Rendering",
+  security: "Security",
+  seo: "SEO",
+  "service-worker": "Service Worker",
+  storage: "Storage",
+  svg: "SVG",
+  testing: "Testing",
+  ux: "UX",
+  "virtual-reality": "Virtual Reality",
+  "web-assembly": "Web Assembly",
+  "web-bundles": "Web Bundles",
+  webxr: "WebXR",
+};
+
+module.exports = {
+  tags,
+  tagDetails: (tag) => {
```

Comment:
Rather than adding this as a function to the exported object, I'd suggest computing all of these values ahead of time and adding them to the exported object. That way we only pay the cost once, versus invoking the function multiple times.